src/common/utility_win.cpp

```diff
@@ -74,6 +74,40 @@ static void setupFavLink_private(const QString &folder)
         qCWarning(lcUtility) << "linking" << folder << "to" << linkName << "failed!";
 }
 
+static void removeFavLink_private(const QString &folder)
+{
+    const QDir folderDir(QDir::fromNativeSeparators(folder));
```

Comment:
is the call to `QDir::fromNativeSeparators` necessary ?
I would have thought that Qt takes care of that by itself

Reply:
@mgallien well, yes, it works without it, just took a copy-paste from the function that creates favlink